docs/workflows/host.rst

```diff
@@ -0,0 +1,107 @@
+.. _host:
+
+Host and Consume a Docker Repository 
+====================================
+
+This section assumes that you have a repository with content in it. To do this, see the
+:doc:`sync` documentation.
+
+Create a Docker Distribution to serve your Repository Version
+-------------------------------------------------------------
+
+Docker Distributions can be used to serve the Docker registry API
+containing the content in a repository's latest version or a specified
+repository version. 
+
+.. literalinclude:: ../_scripts/distribution.sh
+   :language: bash
+
+Response:
+
+.. code::
+
+    {
+        "_created": "2019-09-05T14:29:51.742086Z",
+        "_href": "/pulp/api/v3/distributions/docker/docker/1b461dac-0839-4049-aa8f-92f8e8f7f034/",
+        "base_path": "test",
+        "content_guard": null,
+        "name": "testing-hello",
+        "registry_path": "localhost:24816/test",
+        "repository": "/pulp/api/v3/repositories/fcf03266-f0e4-4497-8434-0fe9d94c8053/",
+        "repository_version": null
+    }
+
+
+
+Reference: `Docker Distribution Usage <../restapi.html#tag/distributions>`_
+
+Perform a pull from Pulp
```

Comment:
Perform a pull from Pulp and run image in a container? On L86 a container is instantiated.